engines/advancedDetector.cpp

```diff
@@ -634,21 +666,68 @@ static bool getFilePropertiesIntern(uint md5Bytes, const AdvancedMetaEngine::Fil
 			return false;
 	}
 
-	if (!allFiles.contains(fname))
-		return false;
+	Common::ScopedPtr<Common::SeekableReadStream> testFile;
+
+	if (md5prop & kMD5Archive) {
```

Comment:
Did you test the cases when a detection entry contains both files inside archives and plain ones?

Reply:
I did, it works flawlessly :)